tox.ini

```diff
@@ -5,8 +5,9 @@ envlist = flake8, django15, django16, django17, django18
 [testenv]
 setenv =
     DJANGO_SETTINGS_MODULE=test_settings
-    PYTHONPATH = {toxinidir}:{toxinidir}/ajax_select
+    PYTHONPATH = {toxinidir}:{toxinidir}/ajax_select:{toxinidir}/tests
```

Comment:
I saw there was a project root level test directory with "test_fields.py" and "test_models.py" in it, so I thought I would add "test_lookups.py" and add that directory to tox.ini, but maybe this wasn't the right way to do it.  Where are the other tests - did I miss them?